src/main/BeerCard.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+function BeerCard(props) {
+  const { beer, onDeleteFavorite, onSetFavorite, favoriteList } = props;
+  const { id, name, image_url, tagline } = beer;
+  let checkFavorites = () => {
+    return favoriteList.find((item) => {
+      return item.id === id;
+    });
+  };
+  let isFav = checkFavorites(id);
```

Comment:
I guess this way it could be better and easier **and** less code.

```suggestion
  const isFav = favoriteList.find((item) => item.id === id);
```